web_widget_many2one_simple/README.rst

```diff
@@ -61,6 +61,9 @@ Options:
 Usage
 =====
 
+Go to Contacts > Create or edit a record and set "Industry" field (in Sales & Purchase tab).
+Only if you set the correct indstry name (Administrative for example) will it be defined, otherwise it will try to create one with that name.
```

Comment:
```suggestion
Only if you set the correct industry name (Administrative for example) will it be defined, otherwise it will try to create one with that name.
```

Reply:
```suggestion
Only if you set the correct industry name ("Administrative" for example), it will be defined. Otherwise, it will try to create one with that name.
```